contrib/libexecinfo/backtrace.c

```diff
@@ -220,8 +220,10 @@ backtrace_symbols_fmt(void *const *trace, size_t len, const char *fmt)
 	}
 
 	/* Change offsets to pointers */
-	for (size_t j = 0; j < len; j++)
-		((char **)(void *)ptr)[j] += (intptr_t)ptr;
+	for (size_t j = 0; j < len; j++) {
+		ptrdiff_t ptr_offs = (intptr_t)((char **)(void *)ptr)[j];
+		((char **)(void *)ptr)[j] = ptr + ptr_offs;
```

Comment:
```suggestion
		ptrdiff_t ptr_offs = (intptr_t)((char **)ptr)[j];
		((char **)ptr)[j] = ptr + ptr_offs;
```

I don't think the intermediate cast is needed?

Reply:
I don't think it is either, but it's in the upstream code, so I assumed @dstolfa was just reducing the diff relative to upstream by keeping it.

Reply:
I was about to suggest the same earlier, but my assumption is it's there because of the asinine -Wcast-align warnings that come from casting `char *` to something more aligned (given `char`'s special status when it comes to aliasing it _really_ should be silent for `char *`).

Reply:
Yeah the goal was just minimizing the diff, but I'm happy to change it as well.

Reply:
I think it's probably fine to leave it as is.

Reply:
Keeping it sgtm